test/acceptance/features/bindAppToServiceUsingConfigMap.feature

```diff
@@ -1,3 +1,4 @@
+@olm
```

Comment:
I would rather not annotation the whole feature with `olm` - it would mean that it fully depend on existence of OLM, but that is not the case. Hence, please annotation individual scnearios where appropriate. Another drawback of annotating the whole feature  is that any new scenarios will be implicitly tagged as well, although perhaps this is not really required.

Reply:
Took inspiration from https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToMultipleServices.feature#L1 https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToService.feature#L1 https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindKnativeService.feature#L2

Reply:
Can we document this in Good practices also? Probably it would be helpful for future PRs?

Reply:
sure, feel free to raise PR for that.

Reply:
For those having 
```
    Background:
        Given Namespace [TEST_NAMESPACE] is used
        * Service Binding Operator is running
        * PostgreSQL DB operator is installed
```

Backgroud as Postgres DB operator installed, that means that is the background for the feature file, in that case also should we tag scenarios instead of the feature file?

Reply:
that background applies to all scenarios in feature file, so then we can just tag `@olm` the whole feature... not the ideal, but we should fix this later.

cc @pratikjagrut